projects/media-viewer/src/lib/annotations/annotation-set/annotation-create/highlight-create.service.ts

```diff
@@ -18,20 +18,24 @@ export class HighlightCreateService {
   width: number;
   zoom: number;
   rotate: number;
+  allPages: object;
 
   constructor(private toolBarEvents: ToolbarEventService,
               private readonly api: AnnotationApiService,
               private store: Store<fromStore.AnnotationSetState>) {
     this.store.select(fromDocument.getPages)
       .subscribe(pages => {
-        this.height = pages.styles.height;
-        this.width = pages.styles.width;
-        this.zoom = parseFloat(pages.scaleRotation.scale);
-        this.rotate = parseInt(pages.scaleRotation.rotation, 10);
+        if (pages[1]) {
```

Comment:
why do you start with 1 and not 0

Reply:
pages are starting with 1 - this is just to make sure i don't assign undefined